src/views/v2/widgets/events-list/view-more.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Widget: Events List Event
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/v2/widgets/events-list/event.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link http://m.tri.be/1aiy
+ *
+ * @version TBD
+ *
+ * @var WP_Post $view_more_link The URL to view all events.
```

Comment:
This seems like a copy-and-pasta issue:

```suggestion
 * @var string $view_more_link The URL to view all events.
```

Reply:
you're right, thanks.